setup.py

```diff
@@ -57,6 +57,7 @@ def find_version(*file_paths):
     ],
     install_requires=[
         "appdirs",
+        "cartopy",

```

Comment:
Added in https://github.com/hydroffice/hyo2_soundspeed/commit/1deeddcc5912b5eecdff09dd46740bf578f5f4b4